src/app/services/manage-people/manage-people.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { IUser } from 'interfaces/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ManagePeopleService {
+  readonly LEGAL_AGE = 18;
+
+  users: Array<IUser> = [
+    {
+      age: 17,
+      firstName: 'Name1',
+      lastName: 'LastName1'
+    },
+    {
+      age: 18,
+      firstName: 'Name2',
+      lastName: 'lastName2'
+    },
+    {
+      age: 19,
+      firstName: 'Name3',
+      lastName: 'lastName2'
+    }
+  ];
+
+  filterIfHasLegalAge(users?: Array<IUser>) {
```

Comment:
Instead of having `filterIfHasLegalAge(users?: Array<IUser>)`, you should set a fallback empty parameter, I will suggest you adding:

```
filterIfHasLegalAge(users: IUser[] = []): ...
```

And you will avoid if condition that checks if users is declared or not:

```
if (!users) {
  return fakeUsers;
}
```